scripts/docker-entrypoint.sh

```diff
@@ -143,11 +141,22 @@ fi
 #
 if [ -e "$prconfig" ]; then
   echo "Loading prconfig from ${prconfig}...";
-  cp "$prconfig" ${CATALINA_HOME}/webapps/prweb/WEB-INF/classes/
+  cp "$prconfig" /opt/pega/prweb/WEB-INF/classes/
 else
   echo "No prconfig was specified in ${prconfig}.  Using defaults."
 fi
 
+/bin/dockerize -template ${CATALINA_HOME}/webapps/ROOT/index.html:${CATALINA_HOME}/webapps/ROOT/index.html
+
+appContextPath=${PEGA_APP_CONTEXT_PATH}
```

Comment:
Defining an extra variable here seems unnecessary, since we're just echoing the same value on the next line.

Reply:
Removed unnecessary variable.